code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
>Taking down one person is easy.

Hellspray + rapid fire syringe gun. I've seen you do it and you know very well it is extremely effective. To hell with "effectiveness"- there's almost no difference.

>Storage space is finite.

Until you get a bag of holding. Which I've seen you do.

The subjobs thing would be neat, however.